client/packages/views/packages-list.js

```diff
@@ -64,10 +64,15 @@ const PackagesList = ( { packages, dimensionUnit, editable, selected, serviceId,
 	return (
 		<FormFieldset className="wcc-shipping-packages-list">
 			<div className="wcc-shipping-packages-list-header">
-				<FormLegend className="package-actions" />
+				{ editable ? null : <FormLegend className="package-actions" /> }
 				<FormLegend className="package-type">{ __( 'Type' ) }</FormLegend>
 				<FormLegend className="package-name">{ __( 'Name' ) }</FormLegend>
-				<FormLegend className="package-dimensions">{ __( 'Dimensions (L x W x H)' ) }</FormLegend>
+				<FormLegend className="package-dimensions">
+					<span className="package-dimensions-title">{ __( 'Dimensions' ) }</span>
+					{ ' ' }
```

Comment:
Won't this space break RTL? Can we _(CSS newbie warning)_ add a `1 em` margin to achieve the same effect?